shared/AppInsightsCore/src/JavaScriptSDK/BaseCore.ts

```diff
@@ -164,15 +167,27 @@ export class BaseCore implements IAppInsightsCore {
             telemetryItem.ver = "4.0";
         }
 
-        // Process the telemetry plugin chain
-        _this.getProcessTelContext().processNext(telemetryItem);
+        if (_this.areExtensionsInitialized()) {
+            // Release queue
+            if (_this._eventQueue.length > 0) {
+                _arrForEach(_this._eventQueue, (event: ITelemetryItem) => {
+                    _this.getProcessTelContext().processNext(telemetryItem);
```

Comment:
`event`?

Reply:
Nice catch, test were still passing because it only verified the number of events, will update tests as well